extensions/README.md

```diff
@@ -102,3 +102,55 @@ To test extensions for a Simple site in Calypso, sandbox the simple site URL (ex
 If you need to modify the build process, bear in mind that config files are also
 synced to WordPress.com via Fusion. Consult with a Jetpack crew member to ensure
 you test the new build in both environments.
+
+## Good to know when developing Gutenberg extensions
+
+### Don't worry about dependencies
+
+The build takes care of core dependencies for both editor and view -scripts. React, lodash and `@wordpress/*` [dependencies](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md) are externalised and automatically enqueued in PHP for your extension.
+
+Extensions _always_ get [Gutenberg's polyfill scripts](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md#polyfill-scripts) enqueued so you can safely use methods not supported by older browsers such as IE11.
+
+### Jetpack sidebar
+
+Jetpack sidebar is registered on all Gutenberg editor pages and filled using [Slot Fill](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/slot-fill#readme).
+
+Just render to `JetpackPluginSidebar` from anywhere in the code:
+
+```jsx
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
+
+<JetpackPluginSidebar>
+	<PanelBody title={ __( 'My sidebar section', 'jetpack' ) }>
+		<p>Jetpack is Bestpack!</p>
+	</PanelBody>
+</JetpackPluginSidebar>
+```
+
+Remember to be mindful of posts, pages, custom post types and re-usable blocks.
+
+### i18n
+As of 04/2019 `wp.i18n` [doesn't support React elements in strings](https://github.com/WordPress/gutenberg/issues/9846). You will have to structure your copy so that links and other HTML can be translated separately.
+
+Not possible:
+
+> Still confused? Check out <a>documentation</a> for more!
+
+Possible:
+
+> Still confused? <a>Check out documentation for more!</a>
+
+### Colours
+
+- To stay consistent with Gutenberg, your extensions should follow [Gutenberg styles and visuals]. Use Gutenberg colour variables where possible.
+- The build also supports [Muriel colours](https://github.com/Automattic/color-studio) via SASS variables (`$muriel-pink-300`) and CSS custom properties (`var( --muriel-pink-300 )`). Prefer CSS custom properties if possible.
+
+### Icons
+
+Please use [Material icons](https://material.io/tools/icons/?style=outline) and don't rely on icons used in core to avoid visual mixing up.
```

Comment:
@scruffian @thomasguillot maybe you have more insight into this?

Reply:
We've had the go-ahead to use Material icons in our projects. What do you think we need more clarity on?

Reply:
_Why_ Material and not dashicons or other core icons:

> don't rely on icons used in core to avoid visual mixing up.

Do we use only outlined Material icons or are  filled ones ok, too?

Reply:
> Why Material and not dashicons or other core icons

Because that's the Muriel guidelines

Reply:
> Do we use only outlined Material icons or are filled ones ok, too?

We are going with outline